source/gui/addonGui.py

```diff
@@ -270,8 +273,13 @@ def __init__(self, parent):
 		mainSizer.Fit(self)
 		self.SetSizer(mainSizer)
 		self.refreshAddonsList()
-		self.addonsList.SetFocus()
+		self.SetMinSize(mainSizer.GetMinSize())
+		# Historical initial size, result of L{self.addonsList} being (550, 350) as of 1364839447.

```

Comment:
I think this was supposed to be a issue number?

Reply:
> I think this was supposed to be a issue number?

Indeed, but this one dates to 2012.2beta1 and was not linked back then to any issue or PR.

Reply:
Is that number a commit? If so, could you change this to `as of commit 1364839447`

Reply:
Done as of commit fede5a9